<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Medhats-MacBook-Pro.local" name="newLoginScenarios.CYCCScenarios_Stubs.TC034_Stubs_CYCC_FailDataRequestAndStartSessionWith401" tests="6" failures="0" timestamp="15 Jan 2020 16:12:52 GMT" time="178.513" errors="1">
  <testcase name="Step3_userDataResponseWithErrorCode401" time="29.489" classname="newLoginScenarios.CYCCScenarios_Stubs.TC034_Stubs_CYCC_FailDataRequestAndStartSessionWith401"/>
  <testcase name="Step1_LaunchTheAppAndOpenMobileTab" time="12.840" classname="newLoginScenarios.CYCCScenarios_Stubs.TC034_Stubs_CYCC_FailDataRequestAndStartSessionWith401"/>
  <testcase name="Step5_SetUserStartSessionWith200AndValidateGDBR" time="29.785" classname="newLoginScenarios.CYCCScenarios_Stubs.TC034_Stubs_CYCC_FailDataRequestAndStartSessionWith401"/>
  <testcase name="Step2_userStartSessionWithErrorCode401" time="27.119" classname="newLoginScenarios.CYCCScenarios_Stubs.TC034_Stubs_CYCC_FailDataRequestAndStartSessionWith401"/>
  <testcase name="Step6_SetUserDataResponseWith200AndValidateGDBR" time="58.114" classname="newLoginScenarios.CYCCScenarios_Stubs.TC034_Stubs_CYCC_FailDataRequestAndStartSessionWith401">
    <error type="org.openqa.selenium.NoSuchElementException" message="Can&#039;t locate an element by this strategy: By.chained({By.id: Weiter})">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Can't locate an element by this strategy: By.chained({By.id: Weiter})
	at io.appium.java_client.pagefactory.AppiumElementLocator.findElement(AppiumElementLocator.java:126)
	at io.appium.java_client.pagefactory.interceptors.InterceptorOfASingleElement.intercept(InterceptorOfASingleElement.java:59)
	at io.appium.java_client.ios.IOSElement$$EnhancerByCGLIB$$d311658.isDisplayed(<generated>)
	at mobileUtilities.Mobile_CommonActions_Set.scrollDownTo(Mobile_CommonActions_Set.java:183)
	at ios_pages.LoginPage.validateLoginErrorPopupAfterPassWith200_Mobile(LoginPage.java:309)
	at newLoginScenarios.CYCCScenarios_Stubs.TC034_Stubs_CYCC_FailDataRequestAndStartSessionWith401.Step6_SetUserDataResponseWith200AndValidateGDBR(TC034_Stubs_CYCC_FailDataRequestAndStartSessionWith401.java:79)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.TimeoutException: Expected condition failed: waiting for io.appium.java_client.pagefactory.AppiumElementLocator$WaitingFunction@30404dba (tried for 1 second(s) with 500 milliseconds interval)
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:304)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
	at io.appium.java_client.pagefactory.AppiumElementLocator.waitFor(AppiumElementLocator.java:99)
	at io.appium.java_client.pagefactory.AppiumElementLocator.findElement(AppiumElementLocator.java:119)
	... 29 more
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate an element using By.chained({By.id: Weiter})
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Medhats-MacBook-Pro.local', ip: 'fe80:0:0:0:8b1:2460:723f:34ae%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.14.6', java.version: '1.8.0_161'
Driver info: driver.version: IOSDriver
	at io.appium.java_client.pagefactory.bys.builder.ByChained.findElement(ByChained.java:74)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
	at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:57)
	at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
	at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
	at io.appium.java_client.pagefactory.bys.ContentMappedBy.findElement(ContentMappedBy.java:50)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
	at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:57)
	at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
	at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
	at io.appium.java_client.pagefactory.AppiumElementLocator.lambda$0(AppiumElementLocator.java:120)
	at io.appium.java_client.pagefactory.AppiumElementLocator$WaitingFunction.apply(AppiumElementLocator.java:172)
	at io.appium.java_client.pagefactory.AppiumElementLocator$WaitingFunction.apply(AppiumElementLocator.java:1)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
	... 31 more
]]>
    </error>
  </testcase> <!-- Step6_SetUserDataResponseWith200AndValidateGDBR -->
  <testcase name="Step4_ValidteBlockingScreen" time="21.166" classname="newLoginScenarios.CYCCScenarios_Stubs.TC034_Stubs_CYCC_FailDataRequestAndStartSessionWith401"/>
</testsuite> <!-- newLoginScenarios.CYCCScenarios_Stubs.TC034_Stubs_CYCC_FailDataRequestAndStartSessionWith401 -->
